### PR TITLE
Add userspace and personal keymaps for brandonschlack

### PR DESCRIPTION
**Pull request by brandonschlack**
## Description

Add my personal keymaps and userspace to QMK Core. I have benefited greatly from seeing all the user functions and maps, that I would like to contribute my own that I've been working on throughout the last year.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd67/rev2/keymaps/brandonschlack/keymap.c [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/* Copyright 2019 MechMerlin
```
Comment (Spaceman) [inline keyboards/kbdfans/kbd67/rev2/keymaps/brandonschlack/keymap.c:1]:
```suggestion
/* Copyright 2019 Brandon Schlack
```

File: keyboards/kingly_keys/romac/keymaps/brandonschlack/keymap.c [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/* Copyright 2018 Jack Humbert
```
Comment (Spaceman) [inline keyboards/kingly_keys/romac/keymaps/brandonschlack/keymap.c:1]:
```suggestion
/* Copyright 2019 Brandon Schlack
```

File: keyboards/nightmare/keymaps/brandonschlack/keymap.c [hunk 3]
```diff
@@ -0,0 +1,196 @@
+/* Copyright 2019 cfbender
```
Comment (Spaceman) [inline keyboards/nightmare/keymaps/brandonschlack/keymap.c:1]:
```suggestion
/* Copyright 2019 Brandon Schlack
```

File: layouts/community/60_ansi_split_bs_rshift/brandonschlack-split/keymap.c [hunk 4]
```diff
@@ -0,0 +1,95 @@
+/* Copyright 2018 Yiancar
```
Comment (Spaceman) [inline layouts/community/60_ansi_split_bs_rshift/brandonschlack-split/keymap.c:1]:
```suggestion
/* Copyright 2019 Brandon Schlack
```

**Review by Spaceman — CHANGES_REQUESTED** [review]

File: keyboards/mechlovin/adelais/keymaps/brandonschlack/keymap.c [hunk 5]
```diff
@@ -0,0 +1,270 @@
+/* Copyright 2020 Team Mechlovin
```
Comment (Spaceman) [inline keyboards/mechlovin/adelais/keymaps/brandonschlack/keymap.c:1]:
```suggestion
/* Copyright 2019 Brandon Schlack
```

**Review by Spaceman — CHANGES_REQUESTED** [review]

File: keyboards/gray_studio/think65/solder/keymaps/brandonschlack/config.h [hunk 6]
```diff
@@ -0,0 +1,20 @@
+/* Copyright 2019 MechMerlin
```
Comment (Spaceman) [inline keyboards/gray_studio/think65/solder/keymaps/brandonschlack/config.h:1]:
```suggestion
/* Copyright 2019 Brandon Schlack
```

**Review by Spaceman — CHANGES_REQUESTED** [review]

File: layouts/community/65_ansi_blocker_split_bs/brandonschlack-split/keymap.c [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+void matrix_init_keymap(void) {
+};
+
+// Runs once after keyboard initializes
+void keyboard_post_init_keymap(void) {
+};
+
+// Runs constantly in the background, in a loop.
+void matrix_scan_keymap(void) {
+};
```
Comment (drashna) [inline layouts/community/65_ansi_blocker_split_bs/brandonschlack-split/keymap.c:92]:
```suggetion
```

**Review by drashna — CHANGES_REQUESTED** [review]
Could you make sure that all of the `c` and `h` files have a GPL2+ compatible license header?

File: keyboards/coseyfannitutti/discipline/keymaps/brandonschlack/keymap.c [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+ * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┴┬──┴──┬───┼───┼───┤
+ * │Ctrl│ Opt│ Cmd│          Space         │ Cmd │FnPly│Lef│Dow│Rig│
+ * └────┴────┴────┴────────────────────────┴─────┴─────┴───┴───┴───┘
+ */
+[_BASE] = LAYOUT_65_ansi_2_right_mods( \
+    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_HOME, \
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP, \
+    HY_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN, \
+    KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,  \
+    KC_LCTL, KC_LOPT, KC_LCMD,                            KC_SPC,                    KC_RCMD, PLY_FN1,          KC_LEFT, KC_DOWN, KC_RGHT  \
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/discipline/keymaps/brandonschlack/keymap.c:38]:
```suggestion
[_BASE] = LAYOUT_65_ansi_2_right_mods(
    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_HOME,
    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
    HY_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
    KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
    KC_LCTL, KC_LOPT, KC_LCMD,                            KC_SPC,                    KC_RCMD, PLY_FN1,          KC_LEFT, KC_DOWN, KC_RGHT
```

etc.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

Be advised that Maartenwut is rebranding as @evyd13, and the Wonderland (along with all other released Maartenwut boards) will be moving to a `keyboards/evyd13/` directory at the end of this month.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**